unit_tests/test_configuration.py

```diff
@@ -119,3 +121,20 @@ def test_migrate(self):
             fd.write("")
         self.configuration.migrate()
         self.assertThat(paths.LEGACY_BOOTSTRAP_FLAG, Not(FileExists()))
+
+    def test_update_center_ca(self):
+        ca_cert = """-----BEGIN CERTIFICATE-----
+-----END CERTIFICATE-----"""
+        orig_update_center_ca = hookenv.config()["update-center-ca"]
+        ca_cert_file = os.path.join(paths.UPDATE_CENTER_ROOT_CAS,
+                                    "default.crt")
+        try:
+            hookenv.config()["update-center-ca"] = ca_cert
+            self.configuration.set_update_center_ca()
+            print(self.fakes.juju.log)
```

Comment:
I don't think we need this any more - just for testing during debugging?

Reply:
c6bc40e